apps/src/music/views/soundsPanel2.module.scss

```diff
@@ -0,0 +1,193 @@
+@import "color.scss";
+
+.soundsPanel {
+  user-select: none;
+  width: 600px;
+  height: 340px;
+  display: flex;
+  flex-direction: column;
+  position: relative;
+
+  &Top {
+    display: flex;
+    justify-content: space-between;
+    margin: 5px 0;
+    user-select: none;
+
+    .segmentedButtons {
+      span {
+        font-size: 12px !important;
+      }
+    }
+  }
+
+  &Body {
+    display: flex;
+    flex-direction: row;
+    overflow: hidden;
+    flex: 1;
+    gap: 5px;
+
+    .leftColumn {
+      background-color: $neutral_dark;
+      width: calc(40% - 00px);
+      overflow-y: scroll;
+    }
+
+    .rightColumn {
+      background-color: $neutral_dark;
+      width: calc(67% + 00px);
+      overflow-y: scroll;
+      display: grid;
+      grid-template-columns: calc(33.3333% - 0px) calc(33.3333% - 0px) calc(33.3333% - 0px);
+      position: relative;
+
+      &FullWidth {
+        width: 100%;
+        grid-template-columns: 20% 20% 20% 20% 20%
+      }
+    }
+  }
+
+  .folderRow {
+    padding: 6px 0;
+    margin: 1px 1px 6px 1px;
+    overflow: hidden;
+    cursor: pointer;
+
+    &:hover {
+      background-color: $neutral_dark90;
+    }
+
+    &:focus-visible {
+      outline: $neutral_light solid 1px;
+    }
+
+    &Selected {
+      background-color: $neutral_dark70;
+    }
+
+    &:first-child {
+      margin-top: 1px;
+    }
+    &:not(:first-child) {
+      margin-top: 14px;
+    }
+
+    &Left {
+      width: 55px;
+      height: 55px;
+      margin: 0 6px 0 4px;
+      float: left;
+    }
+
+    &Middle {
+      width: calc(100% - 75px - 58px);
+      float: left;
+
+      &Name {
+        font-size: 16px;
+      }
+
+      &SubTitle {
+        font-size: 11px;
+        line-height: 1.2em;
+      }
+    }
+
+    &Right {
+      float: right;
+      width: 43px;
+      margin-right: 15px;
+    }
+  }
+
+  .soundEntry {
+    overflow: hidden;
+    height: 110px;
+    font-size: 14px;
+    margin: 4px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    cursor: pointer;
+    box-sizing: border-box;
+    flex-direction: column;
+    border-radius: 5px;
+
+    &Selected {
+      background-color: $neutral_dark70 !important;
```

Comment:
Why is the `!important` needed?

Reply:
It was to avoid the hover overriding the selected color, but I've now found another way that seems to work.